website/docs/api/transformer.md

```diff
@@ -158,7 +158,7 @@ applied to the `Doc` in order. Both [`__call__`](/api/transformer#call) and
 | `batch_size`   | The number of documents to buffer. Defaults to `128`. ~~int~~ |
 | **YIELDS**     | The processed documents in order. ~~Doc~~                     |
 
-## Transformer.begin_training {#begin_training tag="method"}
+## Transformer.initialize {#initialize tag="method"}
```

Comment:
We need to make a corresponding update to https://github.com/explosion/spacy-transformers/blob/master/spacy_transformers/pipeline_component.py

Reply:
✅ https://github.com/explosion/spacy-transformers/commit/f61302482e9d580c442e9622f76ca373cac2cb43